src/interfaces/swig/shogun.i

```diff
@@ -172,6 +172,13 @@ namespace shogun
 			$self->put(tag_t, value);
 	}
 
+	/* get method for string only available from the interfaces to disambiguate
```

Comment:
no need for this comment... everything in here is only available from the interfaces :)

Reply:
I've updated the comment such to explain why we have `get_string` defined here and not as a `%template` directive.